deploy/olm-catalog/bundle/manifests/migration.openshift.io_migclusters.yaml

```diff
@@ -25,6 +27,7 @@ spec:
     listKind: MigClusterList
     plural: migclusters
     singular: migcluster
+  preserveUnknownFields: false
```

Comment:
@jmontleon this was part of the auto-generated manifest out of controller-runtime in mig-controller. I am not sure if this is safe to merge.

I think _lack of this field previously_ was what allowed us to get this far in the dev process on https://github.com/konveyor/mig-controller/pull/954 without running into a CRD error.

This might be related to some changes @shawn-hurley made a while back to CRD generation.